edb/schema/delta.py

```diff
@@ -3405,6 +3405,11 @@ def _cmd_tree_from_ast(
         cmd = super()._cmd_tree_from_ast(schema, astnode, context)
         assert isinstance(cmd, AlterObject)
 
+        # idk how, but somehow there could be an instance
```

Comment:
overloaded functions declared in SDL have `CreateObject` asts (with flags indicating that they can be alters too)